### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jrwpatterson**
<details>
  <summary>📦 Published PR as canary version: <code>1.0.3-canary.2.01eaae6a50a6e4c1044e7f822ba30be0b7aedcdc.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @loanmarket/eslint-config-loanmarket@1.0.3-canary.2.01eaae6a50a6e4c1044e7f822ba30be0b7aedcdc.0
  npm install @loanmarket/common-prettier@1.0.3-canary.2.01eaae6a50a6e4c1044e7f822ba30be0b7aedcdc.0
  # or 
  yarn add @loanmarket/eslint-config-loanmarket@1.0.3-canary.2.01eaae6a50a6e4c1044e7f822ba30be0b7aedcdc.0
  yarn add @loanmarket/common-prettier@1.0.3-canary.2.01eaae6a50a6e4c1044e7f822ba30be0b7aedcdc.0
  ```
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
\ No newline at end of file
+# Common-Quality
+
+## What is this library?
+
+This will be where all the quality systems for all new React Projects go so there is a standard system for all librarys and a uniform code environment
+
+## How do I use the components for this library
+
+You will need to install the peer dependencies
```
Comment (martybarber) [inline README.md:9]: Are you missing the command here?

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+``
+
+Then create a .eslintrc.yml
```
Comment (martybarber) [inline README.md:12]: Couldn't we do this ourselves with a post install step?

Comment (jrwpatterson) [reply README.md:12]: I guess... but seems like a pain for a one off every now and then

Comment (jrwpatterson) [reply README.md:12]: let me fiddle

Comment (martybarber) [reply README.md:9]: up to you

File: README.md (hunk 1 above)
Comment (jrwpatterson) [reply README.md]: yup saw that...

File: README.md (hunk 2 above)
Comment (jrwpatterson) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (jrwpatterson) [reply README.md]: done

**Review by martybarber — APPROVED** [review]

**Pull request merged**